modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -0,0 +1,243 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-log-forwarding.adoc
+
+[id="cluster-logging-log-forwarding-configure_{context}"]
+= Configuring log forwarding
+
+To configure log forwarding, edit the Cluster Logging Custom Resource (CR) to add the `clusterlogging.openshift.io/logforwardingtechpreview: enabled` annotation and create a Log Forwarding Custom Resource to specify the outputs, pipelines, and enable log forwarding. 
+
+If you enable log forwarding, any undefined source types will be dropped. For example, if you specified a pipeline for the `logs.app` 
+source type, but did not specify a pipeline for the `logs.infra`type, `logs.infra` are dropped.
+
+.Procedure
+
+. Edit the Cluster Logging Custom Resource (CR) in the `openshift-logging` project: 
++
+----
+$ oc edit ClusterLogging instance
+----
+
+. Set the `clusterlogging.openshift.io/logforwardingtechpreview` annotation to `enabled`.
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  annotations:
+    clusterlogging.openshift.io/logforwardingtechpreview: enabled <1>
+  name: "instance"
+  namespace: "openshift-logging"
+----
+<1> Enables and disables log forwarding. Set to `enable` to use log forwarding. Set to `disabled` to use the {product-title} Elasticsearch logstore.
```

Comment:
If users want to use the clusterlogging managed Elasticsearch as the log store, no need to set `annotations` in the clusterlogging instance.

Reply:
Correct.  Under the hood we are using the same mechanism but because this is a TP feature we require users to explicitly opt it, thus the annotation